docs/how-to/multi.md

```diff
@@ -0,0 +1,45 @@
+# Deploy an app to multiple clouds and/or regions
```

Comment:
I think it would really be worth it to have a single paragraph right below this title that sells up the ability of Anycloud to deploy a singular logical cluster to multiple regions and/or multiple clouds at the same time, and that it will always try to keep at least one running server in each region/cloud defined.